lib/rules/component-name-in-template-casing.js

```diff
@@ -100,10 +153,18 @@ module.exports = {
           })
         }
       }
-    }, {
-      Program (node) {
-        hasInvalidEOF = utils.hasInvalidEOF(node)
-      }
-    })
+    },
+    Object.assign(
```

Comment:
`Spread in object literals` is supported from Node.js 8.3.0, so I think that it can not be used.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Browser_compatibility

Reply:
Oh!

Reply:
@ota-meshi https://node.green/#ES2018-features-object-rest-spread-properties 8.2.1